src/confluent_kafka/schema_registry/json_schema.py

```diff
@@ -45,7 +45,7 @@ def __exit__(self, *args):
 
 class JSONSerializer(Serializer):
     """
-    Serializes Python objects to JSON with Confluent Schema Registry framing.
+    Serializer that produces JSON formatted data with Confluent Schema Registry framing.
```

Comment:
"produces" is unfortunate wording here since I don't think it actually Kafka-produces anything.
I liked the previous version better.

Reply:
I will change 'produces' to 'outputs'.

I was trying to avoid referring to what the serializer takes as input, as it gets complex to explain (and is explained below).